version-control/tutorial/README.md

```diff
@@ -0,0 +1,13 @@
+# Super Sandwich Survey
+
+Sandwiches are a staple part of many people’s diets, particularly in western culture. However, little data exists on sandwich eating habits today. This repository will be used as a reference guide for anyone taking part in the super sandwich survey. 
+
+Thank you for taking part in the super sandwich survey. Your responses will contribute to a larger resource on people’s experiences with sandwiches. Please read the instructions file to learn more information about how to interact with the sandwich survey and how to log a new sandwich that is not currently in the sandwich database.
+
+---
+
+- It is strongly encouraged to have a README.md file in every git repository. A README.md file gives an overview of what the project it is, and (when on github.com) how other people can get involved, and who maintains and contributes to the project. We'll learn more about these best practices in lessons 2 & 3. 
```

Comment:
I would remove this and the following paragraph. I find it a little confusing that there are two "view points" - one setting up the story of the "experiment" and one explaining what this document is. There'll be more instruction on this later in the workshops, so I think it doesn't need to be written down here, but you can give a verbal explanation of what a README is when you tell people about the files.

Reply:
Yeah I went back and forth on that as well - I will remove these "instructor" points from the README